Makefile

```diff
@@ -84,13 +80,16 @@ regenerate:
 	@$(MAKE) generate
 
 deploy-image: image
-	hack/deploy-image.sh
+	IMAGE_TAG=$(IMAGE_TAG) hack/deploy-image.sh
 
 deploy:  deploy-image deploy-elasticsearch-operator
-	hack/deploy.sh
-
-deploy-no-build:  deploy-elasticsearch-operator
-	hack/deploy.sh
+	LOCAL_IMAGE_CLUSTER_LOGGING_OPERATOR_REGISTRY=127.0.0.1:5000/openshift/cluster-logging-operator-registry \
```

Comment:
i'm also not sure how this works?  you're going to end up with a pod on your cluster that's trying to run the image `127.0.0.1:5000/openshift/cluster-logging-operator-registry` which won't be pullable by the cluster.

Reply:
oh i guess you're just using this value for the build+push, not for what the pod runs, so that part makes sense at least.